client/src/styles.scss

```diff
@@ -32,12 +32,12 @@ $my-primary: mat.define-palette(mat.$indigo-palette, 500);
   }
   .drawer-list-item-active {
     background-color: change-color(
-      mat.get-color-from-palette($primary),
+      white, //looks nice
```

Comment:
I'd drop the "looks nice" comment – that really doesn't add anything useful.

Reply:
Also, should we be using named colors (like `$primary`) from our CSS, i.e., creating a named color (that is just `white`) and referring to that name here?